proto/gds/models/v1/models.proto

```diff
@@ -50,6 +50,28 @@ message GDSExtraData {
     // Temporary: verification token for light weight authentication for verification
     // TODO: replace with admin API that uses authentication
     string admin_verification_token = 1;
+
+    // Change log which represents previous verification state changes made to a VASP
+    repeated VerificationChangeLogEntry verification_change_log = 2;
```

Comment:
What would you think about calling this `AuditLog`? Although primarily this will be used for state changes, I think it would also be used for actions that do not change the state. In the example on line 69 - "resend emails" would look like:

{
  "timestamp": "2021-09-28T07:23:43Z",
  "previous_state:" "EMAIL_VERIFICATION",
  "current_state": "EMAIL_VERIFICATION",
  "description": "resent verification emails to legal and billing contacts",
  "source": "tinker@trisa.io"
}

E.g. we would record all actions relative to the state of the review for the purposes of ensuring that the VASP state was maintained correctly.

Reply:
Good suggestion, I think that name fits better.